<use f='llvm/llvm/utils/unittest/googletest/src/gtest-internal-inl.h' l='803' u='r' c='_ZNK7testing8internal12UnitTestImpl16catch_exceptionsEv'/>
<use f='llvm/llvm/utils/unittest/googletest/src/gtest-internal-inl.h' l='810' u='w' c='_ZN7testing8internal12UnitTestImpl20set_catch_exceptionsEb'/>
<dec f='llvm/llvm/utils/unittest/googletest/src/gtest-internal-inl.h' l='920' type='bool'/>
<use f='llvm/llvm/utils/unittest/googletest/src/gtest.cc' l='4353' u='w' c='_ZN7testing8internal12UnitTestImplC1EPNS_8UnitTestE'/>
<offset>3968</offset>
<doc f='llvm/llvm/utils/unittest/googletest/src/gtest-internal-inl.h' l='918'>// The value of GTEST_FLAG(catch_exceptions) at the moment RunAllTests()
  // starts.</doc>
